app/backend/migrations/20220125150262-add-statuses.js

```diff
@@ -0,0 +1,56 @@
+
+const statusLabels = ['Unsorted', 'To do', 'In progress', 'Done'];
```

Comment:
Why do we need `Unsorted` status? I thought that `unsorted` means `status is null`

Reply:
I didn’t get it. Will remove Unsorted status then

Reply:
done